tools/certissuer/lego.go

```
@@ -0,0 +1,134 @@
+	svc := &certIssuer{}
+
+	for _, provider := range providers {
+		if err = client.Challenge.SetDNS01Provider(provider); err != nil {
+			return nil, err
+		}
+	}
+
+	return svc, nil
+}
+
+func (cl *certIssuer) run() {
```

golangci-lint (1.64.8)

func `(*certIssuer).run` is unused

(unused)